app/src/androidTest/java/org/kiwix/kiwixmobile/intro/IntroFragmentTest.kt

```diff
@@ -30,6 +32,7 @@ class IntroFragmentTest : BaseActivityTest<KiwixMainActivity>() {
   @Test
   fun viewIsSwipeableAndNavigatesToMain() {
     runOnUiThread { activityRule.activity.navigate(R.id.introFragment) }
+    BaristaSleepInteractions.sleep(TestUtils.TEST_PAUSE_MS.toLong())
```

Comment:
we can probably delete this, we should probably even go back to ignoring this test because it is so troublesome but that is another ticket